docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+
+### Deleting a question : `deleteq`
+
+Deletes a question in the question list.
+Once a question has been addressed, you can delete it from the list.
+
+Format: `deleteq INDEX`
+
+* The index refers to the index number shown in the displayed question list.
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `deleteq 1` deletes the first question from the question list
+
+### Tutorials
+Tutorials contain a `group number`, `content` to be taught, and its `date and time`. 
+
+
+### Adding a tutorial : `addtut`
+
+Adds a tutorial to the tutorial list.
+
+Format: `addtut g/GROUP_NUMBER c/CONTENT t/DATE TIME`
+
+* The format of the date must be in the YYYY-MM-DD.
+* The format of the time must be in 24h format HHmm.
+
+Example:
+
+* `addtut g/T08 c/UML diagrams t/2022-10-01 1400`
+
+### Deleting a tutorial : `deletetut`
+
+Deletes a tutorial in the tutorial list.
+
+Format: `deletetut INDEX`
+
+* Deletes the tutorial at the specified INDEX.
+* The index refers to the index number shown in the displayed tutorial list.
+* The index must be a positive integer 1, 2. 3, … .
+*
+
+Example:
+
+* `deletetut 1`deletes the first tutorial from the tutorial list
+
+### Marking a tutorial: `marktut`
+
+Marks content in the tutorial as done.
+When the tutorial is over, you can indicate on the user interface that it is done.
+
+Format: `marktut INDEX`
+
+* Marks the tutorial at the specified `INDEX`.
+* The index refers to the index number shown in the displayed tutorial list.
+* The index must be a positive integer 1, 2, 3, ... .
+
+Example:
+
+* `marktut 1` marks the first tutorial from the tutorial list as done.
+
+### Unmarking a tutorial: `unmarktut`
+
+Marks content in the tutorial as undone.
+If a tutorial was mistakenly marked as done, you can undo it using this feature.
+
+Format: `unmarktut INDEX`
+
+* Marks the tutorial at the specified `INDEX`.
+* The index refers to the index number shown in the displayed tutorial list.
+* The index must be a positive integer 1, 2, 3, ...
+
+Example:
+
+* `unmarktut 1` marks the first tutorial from the tutorial list as undone.
+
+### Exiting the program : `exit`
+
+Exits the program.
+
+Format: `exit`
+
+### Saving the data
+
+SETA data are saved in the hard disk automatically after any command that changes the data. There is no need to save
+manually.
+
+### Editing the data file
+
+SETA data are saved as a JSON file `[JAR file location]/data/seta.json`. Advanced users are welcome to update data
+directly by editing that data file.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+If your changes to the data file makes its format invalid, SETA will discard all data and start with an empty data file at the next run.
+</div>
+
+### Archiving data files `[coming in v2.0]`
+
+_Details coming soon ..._
+
+
```

Comment:
Add the clear feature here too because we have it in still!

Reply:
done